ui/static/ui/js/learningresources/learning_resource_panel.jsx

```diff
@@ -167,6 +180,8 @@ define("learning_resource_panel", ['react', 'jquery', 'lodash',
           thiz.setState({
             message: "Form saved successfully!"
           });
+          // user can close panel
+          thiz.props.markDirty(false);
```

Comment:
`descriptionOriginal` and `vocabulariesAndTermsOriginal` should be updated here, it is not sufficient to just set the dirty flag to false. Please add the `updateDirty` method as described below, then replace this call with changes to `descriptionOriginal` and `vocabulariesAndTermsOriginal` and call `updateDirty`